Geo-Shader-Sphere-100-repulsion_max_128.py

```diff
@@ -174,25 +185,45 @@ def vtkShaderCallback3(caller, event, calldata=None):
             program.SetUniform3f("vertices[%d]"%(i), vertices[i].tolist())
         for j in range(330):
             program.SetUniform3f("order[%d]"%(j), [final_vertices_list[j], 0, 0])
-        program.SetUniformi("delay", 220)
+        program.SetUniformi("delay", 165)
 
 
 mapper3.AddObserver(vtk.vtkCommand.UpdateShaderEvent,vtkShaderCallback3)
 
+@vtk.calldata_type(vtk.VTK_OBJECT)
+def vtkShaderCallback4(caller, event, calldata=None):
+    program = calldata
+    if program is not None:
+        for i in range(100):
+            program.SetUniform4f("vertices[%d]"%(i), vertices[i].tolist())
```

Comment:
Why is this SetUniform4f? We have 3 everywhere else. This raises an error as well. Using the code by setting this to 3.